jena-project/pom.xml

```diff
@@ -74,13 +75,14 @@
     -->
     <ver.httpcore-osgi>${ver.httpcore}</ver.httpcore-osgi>
     <ver.httpclient-osgi>${ver.httpclient}</ver.httpclient-osgi>
+    
+    <ver.shiro>1.4.0</ver.shiro>
 
-    <ver.commons-codec>1.10</ver.commons-codec>
     <ver.lucene>6.4.1</ver.lucene>
 
     <ver.elasticsearch>5.2.2</ver.elasticsearch>
     <!-- elasticsearch uses this by reflection -->
-    <ver.log4j2>2.7</ver.log4j2>
+    <ver.log4j2>2.9.1</ver.log4j2>
 
```

Comment:
Given the warning in the comment, sticking to 2.7 is safer (it is what ES depends on).